tests/features/support/env.rb

```diff
@@ -1,5 +1,6 @@
 require 'rubygems'
 require 'selenium-cucumber'
+require 'pry'
```

Comment:
do we require pry at runtime? since we have mentioned it as development dependency in gemspec.